### PR TITLE
Topic 4

### PR DESCRIPTION
**Pull request by darachtcd2**
final version of feature1 and a new line

### CODE REVIEW

File: dtcd2.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ ok, here's a second line
 some dev
 more dev
 and some more
+a line
```
Comment (darachtcd) [inline dtcd2.txt:6]: just one line?

File: feature1_file.txt [hunk 2]
```diff
@@ -4,3 +4,4 @@ dev
 dev
 a
 cd
+really finish this
```
Comment (darachtcd) [inline feature1_file.txt]: I don't think this is really finished
you have to really really finish it

File: feature1_file.txt [hunk 3]
```diff
@@ -4,3 +4,4 @@ dev
 dev
 a
```
Comment (darachtcd) [inline feature1_file.txt:5]: this is fine

File: dtcd2.txt (hunk 1 above)
Comment (darachtcd2) [reply dtcd2.txt:6]: I think that's enough

File: feature1_file.txt (hunk 2 above)
Comment (darachtcd2) [reply feature1_file.txt]: really really finished

Comment (darachtcd) [reply feature1_file.txt]: ok

File: feature1_file.txt [hunk 4]
```diff
@@ -4,4 +4,4 @@ dev
...
 cd
-really finish this
+really really finish this
```
Comment (darachtcd) [inline feature1_file.txt:7]: ok

File: dtcd2.txt (hunk 1 above)
Comment (darachtcd2) [reply dtcd2.txt:6]: just add one more line

Comment (darachtcd) [reply dtcd2.txt:6]: just add one more line

File: dtcd2.txt [hunk 5]
```diff
@@ -3,3 +3,5 @@ ok, here's a second line
...
+another line
```
Comment (darachtcd) [inline dtcd2.txt:7]: fine, merging

**Review by darachtcd — APPROVED** [review]

**Pull request merged**